exercises/anagram/test/test_anagram.c

```diff
@@ -4,11 +4,35 @@
 #include "vendor/unity.h"
 #include "../src/anagram.h"
 
+// Empty vector for when expected output is nothing
+struct Vector empty = {
+   NULL,
+   0
+};
+
 int stringcmp(const void *a, const void *b)
 {
    return strcmp(a, b);
 }
 
+// Helper function to verify the correct output
+void check_anagrams(char *in, struct Vector vin, struct Vector expected)
```

Comment:
I just realized that nothing in any of our tests is marked `static`. This function in particular has a name that might come up in someone's solution and get the linker confused. How about making at least the helper functions and helper function data (ie: `empty`) `static`?